mods/d2k/cursors.yaml

```diff
@@ -1,7 +1,7 @@
 ShadowIndex: 1
 
 Palettes:
-	mouse: d2k.pal
+	mouse: palette.bin
```

Comment:
These were uppercase in the disc and uppercase in my content folder as well. You're breaking Linux here.